### DISCUSSION TITLE
Which environment variables are available despite from MONGODB_URI?

### DISCUSSION THREAD

**Discussion by DaniDeer** [percona/mongodb_exporter#889] [Q&A] [answered]
Are the flags for docker run as described [in the References](https://github.com/percona/mongodb_exporter/blob/main/REFERENCE.md) are also available as environment variables? e.g. for configuring the debug level ($LOG_LEVEL).

Or is it limited to only $MONGODB_URI?

I tested $LOG_LEVEL debug but it seems it does not affect the exporter config.

**Comment by idoqo** [comment, marked as answer]
At the moment, only `MONGODB_USER`, `MONGODB_PASSWORD`, and `MONGODB_URI` are available via environment variables sadly. Log levels have to be changed with a direct command flag
